src/ShowPage/ShowPage.css

```diff
@@ -0,0 +1,17 @@
+.show-page-container {
+  background-color: #e5e6eb;
+  padding: 3em;
+}
+
+.show-page-image {
+  height: 25em;
+}
+
+.genres {
+  display: flex;
+}
+
+.genres > p {
```

Comment:
Thanks for teaching me about using `>` to target child elements.  Super neat!
We still have to work on getting those genre values on the same line as the Genres description on the `ShowPage`... perhaps we could just evaluate `genre` in our mapping method so that it is no longer returned as part of the `<p>` element.